src/community/propose-a-component-or-pattern/index.md.njk

```diff
@@ -36,9 +36,6 @@ At the meeting, the working group will review your proposal and decide if it is
 
 After the meeting, the Design System team will let you know the decision and recommendations, if there are any.
 
-If the working group agrees your proposal is useful and unique, the Design System team will mark it as 'to do' on the [community backlog](/community/backlog/). 
+If the working group agrees your proposal is useful and unique, the Design System team will move it to the next step and mark it as 'Needs more detail' on the [Community Backlog project board on GitHub](https://github.com/alphagov/govuk-design-system-backlog/projects/1). 
```

Comment:
I think GOV.UK convention is for sentences to be no longer than 25 words.

```suggestion
If the working group agrees your proposal is useful and unique, the Design System team will:

- move the proposal to the next step
- mark it as 'Needs more detail' on the [Community Backlog project board on GitHub](https://github.com/alphagov/govuk-design-system-backlog/projects/1)
```

Bonus question: are "next step" and "'Needs more detail'" the same thing?